src/sentry/workflow_engine/endpoints/validators/base/action.py

```diff
@@ -66,13 +66,17 @@ def validate(self, attrs: dict[str, Any]) -> dict[str, Any]:
 
         attrs = super().validate(attrs)
 
+        if not Action.Type(attrs["type"]).is_integration() and attrs.get("integration_id"):
+            raise serializers.ValidationError(
+                f"Integration ID is not allowed for action type {attrs["type"]}"
+            )
```

Comment:
should the order of ops here be:
- check for the organization
- validate with super
- then this integration check?

My thought is we should order them based on speed + order of operations, so we can fail quickly and do the least amount of work.